.github/ISSUE_TEMPLATE/hosting.md

```diff
@@ -0,0 +1,57 @@
+---
+name: hosting
+about: Wanting to host the meetup? Answer the following questions
+---
+
+Thank you for wanting to host the Ruby Usergroup Hamburg!
+Here we have collected some information regarding hosting the Usergroup.
+Additionally, it would be great if you could answer the questions at the end of the issue. This will help us to have a good overview
+
+# FAQ
+
+## When can I host the meetup
+We meet every **second Wednesday** of the month
+Any [event](https://hamburg.onruby.de/#events) where no location is named still requires one
+
+## How much space do we need?
+There have been **17** Rubyists signed up on average for the latest events and there were about **1** to **2** presentations held. 
```

Comment:
instead of copying this here it might make more sense to just link to the respective page on onruby. especially as most of this is duplicated in the checkbox section.

Reply:
Yes you are right 
I removed the FAQ part and created an issue at the on_ruby repo
https://github.com/phoet/on_ruby/issues/651

Yet I think the issue template for the hosting might be a good idea for us. I believe it can give some guidance to someone who wants to host the meetup and we can link it to and meetup issues as well.